tokio-macros/src/lib.rs

```diff
@@ -79,7 +79,7 @@ pub fn test(_attr: TokenStream, item: TokenStream) -> TokenStream {
         #(#attrs)*
         fn #name() #ret {
             let mut rt = tokio::runtime::current_thread::Runtime::new().unwrap();
-            rt.block_on_async(async { #body })
+            rt.block_on(async { #body })
```

Comment:
change seems unrelated?

Reply:
Yeah, it should probably be a separate PR (like #1198).

Reply:
I'm ok bundling this in the PR.